platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/event/preconsensus/PcesWriterTests.java

```diff
@@ -663,7 +670,7 @@ void discontinuityTest(@NonNull final AncientMode ancientMode) throws IOExceptio
     @MethodSource("buildArguments")
     @DisplayName("Advance Non Ancient Boundary Test")
     void advanceNonAncientBoundaryTest(@NonNull final AncientMode ancientMode) throws IOException {
-        final Random random = RandomUtils.getRandomPrintSeed();
+        final Random random = RandomUtils.getRandomPrintSeed(284520236160395337L); // TODO
```

Comment:
TODO

Reply:
Removed TODO and seed. I fixed the bug in my previous commit but forgot to remove the seed I was using for debugging.

Something worth bringing up with the rest of the team -- it would be nice of the existence of the string `TODO` caused the CI pipeline to fail.